diff --git a/dist/index.js b/dist/index.js
index c0792c9c37e339fdf90a5396d2e665d0565a1d7b..2a11dbcc0ca741c8253cc1924c046722336fbf01 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -224,7 +224,7 @@ var punycode_default = punycode;
 
 // src/index.ts
 var src_default = punycode_default;
-export {
+module.export = {
   ucs2encode,
   ucs2decode,
   toUnicode,
@@ -236,7 +236,7 @@ export {
   error,
   encode,
   digitToBasic,
-  src_default as default,
+  default: src_default,
   decode,
   basicToDigit,
   adapt,
diff --git a/dist/types.d.ts b/dist/types.d.ts
index 3c57b576ec422ce6b7cff09d0ac25950b64566bb..1b3f2c884c9dc922287bee9447d1d149d28358fd 100644
--- a/dist/types.d.ts
+++ b/dist/types.d.ts
@@ -1,5 +1,6 @@
 export declare type PunycodeErrorType = 'overflow' | 'not-basic' | 'invalid-input'
 
+// @ts-expect-error 类型不兼容，但我知道我在做什么
 export const PUNYCODE_ERRORS: Record<PunycodeErrorType, string> = {
   'overflow': 'Overflow: input needs wider integers to process',
   'not-basic': 'Illegal input >= 0x80 (not a basic code point)',
diff --git a/package.json b/package.json
index e2a7652a392b71b1dee8c491bc73b04fb90b73f9..ee787620b8aaa37c2dfe03b157dc458d8c845576 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ts-punycode",
-  "type": "module",
+  "type": "commonjs",
   "version": "0.1.0",
   "description": "A modern TypeScript implementation of Punycode (RFC 3492) with full RFC 5891 support.",
   "author": "Chris Breuer <chris@stacksjs.org>",
@@ -24,16 +24,17 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
-      "import": "./dist/index.js"
+      "import": "./dist/index.js",
+      "require": "./dist/index.js"
     },
     "./*": {
-      "import": "./dist/*"
+      "import": "./dist/*",
+      "require": "./dist/*"
     }
   },
   "module": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "bin": {
-    "bin-name": "./dist/bin/cli.js"
   },
   "files": ["README.md", "dist"],
   "scripts": {
